src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menAges = (century)
+    ? people
+      .filter(person => Math.ceil(person.died / 100) === century
+        && person.sex === 'm')
+    : people
+      .filter(person => person.sex === 'm');
```

Comment:
DRY with `sex`, let's use ternary inside `filter`

Reply:
i did ternary inside filter, but i still have a filter in ternary (